src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed');
```

Comment:
> That being said, lets see what @damithc sir has to say about this suggestion before you implement it.

I have no strong opinions about this, as I'm not a JS expert. So, I'll let the other senior devs decide 😛